e2e/helpers/case_helper.js

```diff
@@ -83,6 +91,17 @@ const populateWithData = async (caseId, data) => {
   return post(url, data, headers);
 };
 
+const getCaseData = async (caseId) => {
+  const authToken = await getAuthToken();
```

Comment:
Couldn't we use currently logged in token? Use the system user as last resort? Aren't the users perms enough?

Reply:
i think sys user is better, we want as much filed as possible and not restricted to the actual user.